how-to/getting-started-with-r/getting_started_with_r.md

```diff
@@ -0,0 +1,72 @@
+![R logo](https://www.r-project.org/Rlogo.png)
+
+# Getting started with R
+
+R is a free, open-source high-level programming language and software environment used primarily for statistical computing and graphics. 
```

Comment:
Comma after open-source:
```
R is a free, open-source, high-level programming language
```
Since "free and open-source" is a compound phrase with a very particular meaning, you could also say
```
R is a free and open-source, high-level programming language
```